tests/SDK/Unit/EnvironmentVariablesTraitTest.php

```diff
@@ -111,10 +96,9 @@ public function emptyProvider()
     }
 
     /**
-     * @test
      * @dataProvider booleanProvider
      */
-    public function environmentVariables_bool_get(string $input, bool $default, bool $expected)
+    public function environment_variables_bool_get(string $input, bool $default, bool $expected)
```

Comment:
this needs a test_ prefix since the @test annotation is removed

Reply:
Thank you! This file needed some manual editing as some of the tests had bad docblocks.
Fixed